### ISSUE TITLE
Add skip link

### ISSUE THREAD

**Issue by julien-deramond** [Open-reSource/openresource.dev#16]
### Prerequisites

- [X] I have [searched](https://github.com/Open-reSource/openresource.dev/issues?q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Open-reSource/openresource.dev/blob/main/CONTRIBUTING.md)

### Proposal / Motivation and context

There's not much to describe here, a skip link should be added to the website for obvious reasons.
